pages/common/git-bisect.md

```diff
@@ -0,0 +1,20 @@
+# git bisect
+
+> Use binary search to find the commit that introduced a bug.
+> Git automatically jumps back and forth in the commit graph to progressively narrow down the faulty commit.
+
+- Start a bisect section by passing a known good commit and a known bad commit:
```

Comment:
I was slightly stumped in understanding "known bad commit". How can I know the bad commit if I have to find it in the first place ? Then it dawned on me, that its a commit sufficiently back in time so that the actual commit which introduced the bug is in the range. 

Maybe we can rephrase it as - "Start a bisect section by passing a start and end commit which contains the commit we want to find".